snapcraft/main.py

```diff
@@ -209,9 +208,7 @@ def main(argv=None):
     except Exception as e:
         if args['--debug']:
             raise
-
-        logger.error(textwrap.fill(str(e)))
-        sys.exit(1)
+        sys.exit(e)
```

Comment:
This is a bad merge. I changed these lines after you made your branch.
The correct merged statements would be:
logger.error(str(e))
sys.exit(1)